src/storage/v2/disk/storage.cpp

```diff
@@ -1583,6 +1611,58 @@ void DiskStorage::DiskAccessor::Abort() {
   disk_transaction_ = nullptr;
 
   is_transaction_active_ = false;
+
+  auto *disk_storage = static_cast<DiskStorage *>(storage_);
+
+  uint64_t transaction_id = transaction_.transaction_id.load(std::memory_order_acquire);
+  for (const auto &delta : transaction_.deltas) {
```

Comment:
Can we move this logic to a separate function to simplify Abort() function code?